guides/studio-development.md

```diff
@@ -7,18 +7,11 @@ To run against locally developed Studio:
 - Clone the `cypress-services` repo (this requires that you be a member of the Cypress organization)
```

Comment:
@mschile Meant to commit this to a different branch actually, but this is fine really, it doesn't matter. Just updating instructions.